composer.json

```diff
@@ -129,6 +129,7 @@
         "ext-posix": "Required to use all features of the queue worker.",
         "ext-redis": "Required to use the Redis cache and queue drivers (^4.0|^5.0).",
         "aws/aws-sdk-php": "Required to use the SQS queue driver, DynamoDb failed job storage and SES mail driver (^3.155).",
+        "brianium/paratest": "Required to run tests in parallel (^6.0).",
```

Comment:
This needs adding to the testing component composer.json too.

Reply:
done.